### PR TITLE
android_alarm_manager_plus example update for supporting "null safety"

### PR DESCRIPTION
**Pull request by manishkumar-dsi**
Changes have been done to support null safety.

Tested in  Flutter (Channel stable, 2.8.1, on Microsoft Windows [Version 10.0.19042.630], locale en-IN)

## Description

*Current example is throwing error in latest Flutter. So I have converted this example for supporting null safety. There is no functionality change. Tested on Flutter 2.8.1 on windows and it worked as expected. *

## Related Issues

NA

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process.

- [x ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [ ] I updated the version in `pubspec.yaml` and `CHANGELOG.md`.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

**Comment by vbuberen** [comment]
Thanks for your contribution. Please resolve analyzer issues, which were reported by the CI.

**Review by vbuberen — COMMENTED** [review]
Thanks for your contribution. Please fix issues reported by analyzer first.

**Comment by vbuberen** [comment]
@manishkumar-dsi Ping in case you missed previous comment.

**Comment by manishkumar-dsi** [comment]
Hi @vbuberen ,
Could not get time during weekdays. I will be working on this from today.
Thanks for your review.

**Comment by manishkumar-dsi** [comment]
Hi @vbuberen ,

Please review.

File: packages/android_alarm_manager_plus/example/android/build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.5.31'
+    ext.kotlin_version = '1.5.20'
```
Comment (vbuberen) [inline packages/android_alarm_manager_plus/example/android/build.gradle:2]: Are there any specific reason to rollback Kotlin version? If there are no issues with version which was used before please revert this change.

**Review by vbuberen — COMMENTED** [review]
Looks good to me. Just one small question about Android Kotlin dependency.

**Review by vbuberen — APPROVED** [review]
Thanks again for your contribution to PlusPlugins.
